data/org.freedesktop.UDisks2.xml

```diff
@@ -1344,8 +1344,8 @@
         Gets a file descriptor for the device that is suitable to be
         used for benchmarking the device (transfer rate, access time
         etc.). Note that the file descriptor may be opened with the
-        <literal>O_DIRECT</literal> and <literal>O_SYNC</literal>
-        flags so care must be taken to only perform page-aligned I/O.
+        <literal>O_DIRECT</literal> flag so care must be taken to only perform
+        block-aligned (512 byte aligned) I/O.
```

Comment:
Please don't remove the mention of the ``O_SYNC`` flag, it's being used too. Also, I think the block size is the underlying device's block size which doesn't necessarily have to be 512 bytes (many modern disks and/or flash drives have 4K blocks). So drop the specific number, please.